showcase/treemap/zoomable-treemap.js

```diff
@@ -0,0 +1,184 @@
+      };
+    });
+  }
+
+  render() {
+    const {modeIndex} = this.state;
+    const options = {
+      mode: MODE[this.state.modeIndex],
+    }
+
+    return (
+      <div className="centered-and-flexed">
+        <div className="centered-and-flexed-controls">
+          <ShowcaseButton
+            onClick={this.updateModeIndex(false)}
+            buttonContent={'PREV MODE'}
+          />
+          <div> {MODE[modeIndex]} </div>
+          <ShowcaseButton
+            onClick={this.updateModeIndex(true)}
+            buttonContent={'NEXT MODE'}
+          />
+        </div>
+
+        <div>
+          <div style={{
+            display: 'flex',
+            alignSelf: 'self-start'
+          }}>
+            { this.state.breadcrumbs.map((crumb, i) => (
+              <div key={crumb} onClick={() => this.sliceTreeFrom(crumb)} >
+                { i !== 0 && <span style={{margin: '6px', opacity: .5, color: 'red'}}>></span> }
+                <span style={{color: 'blue', textDecoration: 'underline', cursor: 'pointer'}}>{crumb}</span>
+              </div>
+              ))
+            }
+          </div>
+          { getContentFunc(this.state.data, leaf => this.sliceTreeFrom(leaf.data.name), options) }
+        </div>
+      </div>
+    );
+  }
+}
+
+function getContentFunc(data, handler, options, node) {
+  /* Filtrate root nodes */
+  if (node && node.depth === 0) return null;
+  if (!data.name) data.name = DEFAUL_ROOT_NAME;
+
+  /* Get label. If label not in leaf we don't wrap lines */
+  const getLabel = d =>
+    <div style={{
+      cursor: 'pointer',
+      paddingLeft: '2px',
+      display: 'flex',
+      whiteSpace: `${d.children ? 'no' : ''}wrap`,
+      flexFlow: `row ${d.children ? 'no' : ''}wrap`
+    }}>
+      <div style={{
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        paddingRight: '4px',
+      }}>{ d.name }</div>
+      <div> { d.value && `[${d.value}]` }</div>
+    </div>;
+
+  const titleHeight = 24;
+  const gap = 8;
+
+  return data
+    ? <div style={{
+      background: 'rgb(100, 100, 100)'
+    }}>
+        <style dangerouslySetInnerHTML={{__html: `
+          .rv-treemap__leaf:hover { background: rgba(255, 255, 255, 0.2) !important }
+        `}} />
+        <Treemap
+          data={data}
+          // getContent={leaf => getContentFunc(leaf.data, handler, options, leaf)}
+          getLabel={getLabel}
+          onLeafClick={handler}
+          getSize={d => d.value}
+          getColor={d => d.hex}
+          getChildren={d => d.children}
+          height={600 || node.y1 - node.y0 - titleHeight - (gap * 2)}
+          width={1000 || node.x1 - node.x0 - (gap * 2)}
+          titileHeight={titleHeight}
+          paddingInner={0}
+          animation={false}
+          className='nested-tree-example'
+          colorType='literal'
+          colorRange={['#12939a8f']}
+          padding={gap}
+          margin={0}
+          getColor={colorFromValue}
+          renderMode='DOM'
```

Comment:
Do your changes work in SVG and DOM mode?

Reply:
Yes, changes work in svg mode, but because onLeafClick not trigger in svg mode, navigation looks broken.